webauthn/idlharness.https.window.js

```diff
@@ -14,35 +14,13 @@ idl_test(
     idlArray.add_untested_idls("[Exposed=(Window,Worker)] interface ArrayBuffer {};");
     idlArray.add_objects({
       WebAuthentication: ['navigator.authentication'],
```

Comment:
Illustrating how poorly these tests are maintained, navigator.authentication was removed from the spec something like 2 years ago, when WebAuthentication was integrated into the Credential Management API. So I don't know that this file being useful for anything after the other methods are removed?

Is it possible just to outright rename this file to idlharness-manual.https.window.js?

Reply:
idlharness enumerates definitions of web spec IDL and generates tests about whether the interfaces in the browser match. I'd prefer that they got dusted off than just totally ignored (otherwise they might as well be deleted.)

Reply:
I don't believe navigator.authentication exists on any browser, so removing that makes sense.